packages/rocketchat-ui-sidenav/client/sideNav.html

```diff
@@ -2,9 +2,6 @@
 		<aside class="sidebar sidebar--{{sidebarViewMode}} {{#if sidebarHideAvatar}}sidebar--hide-avatar{{/if}}" role="navigation">
 			{{> sidebarHeader }}
 			{{#if loggedInUser}}
-				{{#if threadingFromSidebar}}
```

Comment:
I absolutely understand you didn't like that button - I don't like it either. But: Is there a replacement for that? I believe there should be an option to start a thread without opening the room first.